### PR TITLE
Add page about the social event

### PR DESCRIPTION
**Pull request by clytaemnestra**

### CODE REVIEW

**Review by hypha — COMMENTED** [review]
Lovely-looking social event! I sort of feel that many people would want to know if food is included or whether they should eat first before heading there.

**Comment by clytaemnestra** [comment]
> Lovely-looking social event! I sort of feel that many people would want to know if food is included or whether they should eat first before heading there.

@hypha thanks for the review! 🙂 I agree - I've added basic info. We don't have the catering confirmed yet, so I'll update it once we know more info. Nevertheless, we'd like to merge this one first and then update it later in a different MR, because we should start selling the tickets. 

Could you have a final look, please? 👁️👁️ I'm not really into games, so I'm open to any suggestions what to add there so it sounds more geeky and it attracts its audience 🙂

**Review by hypha — COMMENTED** [review]
@clytaemnestra, this looks really good to me with the added information. Totally makes sense to add more detailed. catering info later. 

I think your board game part looks good enough already. But I tried to add a bit more flair since you asked. The suggestion below is just in case you want to pick parts of it. 
`We will have a dedicated area for board games! Whether you’re up for an epic night of strategy showdowns or some light-hearted fun, bring your favourite games and form your crews. Experienced gamers and novices alike are all welcome to join!`

File: src/content/pages/social-event.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Located in the heart of Prague, the historic Gabriel Loci ex-monastery is our venue for our main social event featuring:
+* **Board Games Night**: A dedicated area for playing board games.
+* **Chit-Chat with a View**: Enjoy conversations with a stunning night view of Prague.
+* **Chill and Quiet Spaces**: Plenty of area to relax.
+* **Singing and Playing Instruments**: Bring your instruments and join us for a musical evening of singing and playing together!
+
+![](./images/gl.jpg)
+
+# Bring Your Board Games
+Bring your games! We will have an area dedicated to board games and encourage everyone to bring their own. Whether you're an experienced gamer or a novice, you're welcome to join!
```
Comment (clytaemnestra) [inline src/content/pages/social-event.mdx:15]:
```suggestion
We will have a dedicated area for board games! Whether you’re up for an epic night of strategy showdowns or some light-hearted fun, bring your favourite games and form your crews. Experienced gamers and novices alike are all welcome to join!
```

**Comment by clytaemnestra** [comment]
@hypha thanks a lot, I've updated it! 🙂 I'll proceed and merge this MR, as we need the info on the web ASAP. We'll add more details later.

**Pull request merged**